definitions/npm/mersenne-twister_v1.x.x/flow_v0.33.x-/mersenne-twister_v1.x.x.js

```diff
@@ -0,0 +1,18 @@
+declare module 'mersenne-twister' {
+  declare class $npm$MersenneTwister {
```

Comment:
Won't Flow think it was something exported from the module? I didn't want Flow to allow `import {MersenneTwister} from 'mersenne-twister'`.